holland/core/cmdshell.py

```diff
@@ -21,6 +21,11 @@
 
 LOGGER = logging.getLogger(__name__)
 
+holland_conf = '/etc/holland/holland.conf'
+currplatform = sys.platform
+if currplatform.startswith('freebsd'):
+    holland_conf = '/usr/local' + holland_conf
```

Comment:
Should probably use [os.path.join()](https://docs.python.org/3/library/os.path.html#os.path.join) for this and the others.